noxfile.py

```diff
@@ -254,14 +261,14 @@ def migration_test(session):
     files = glob.glob("test_migration/versions/*.py")
 
     # updating the upgrade-script code
-    with open(files[0], "r") as f:
-        script_code = f.read()
+    with open(files[0], "rb") as f:
+        script_code = f.read().decode()
 
     script_code = script_code.replace(
         """def upgrade() -> None:\n    pass""", UPGRADE_CODE
     )
-    with open(files[0], "w") as f:
-        f.write(script_code)
+    with open(files[0], "wb") as f:
+        f.write(script_code.encode())
```

Comment:
Locally I've got an error with rewriting a migration script by a program - some symbols were missing. Rewriting the file, using bytes, fixes the problem.